bundles/org.openhab.ui/web/src/components/widgets/layout/oh-layout-page.vue

```diff
@@ -15,7 +15,7 @@
       </f7-block>
 
       <hr v-if="context.editmode">
-      <f7-block v-if="context.component.slots.masonry && context.component.slots.masonry.length">
+      <f7-block v-if="context.component.slots.masonry && context.component.slots.masonry.length" :style="{ 'z-index': 'auto !important' }">
```

Comment:
I have to check for side effects, but using the regular `style` property would work just as well:
```suggestion
      <f7-block v-if="context.component.slots.masonry && context.component.slots.masonry.length" style="z-index: auto !important">
```
No need to involve Vue.JS if there's no need for it.

Reply:
I will remove this as I left this in by mistake.  I was trying to model this similar to how the z-index is set in the oh-block.vue ( style="{ 'z-index': (context.editmode) ? 100 - context.parent.component.slots.default.indexOf(context.component) : 'auto !important' }" ), but that broke things in edit mode, so I just hardcoded.